### ISSUE TITLE
Tooltip - Add a correct unit test on svg elements

### ISSUE THREAD

**Issue by Johann-S** [twbs/bootstrap#24636]
Related to that conversation : https://github.com/twbs/bootstrap/pull/24606#discussion_r148015459

Maybe @XhmikosR can add a bit of details what's a correct SVG 👍

**Comment by XhmikosR** [comment]
Well, the title cannot be in the circle element.

/CC @patrickhlauke

**Comment by patrickhlauke** [comment]
correct. `title` attribute is invalid in `<svg>` content. which is why i removed the visual test. while it was interesting that our tooltip code worked with invalid `<svg>`, it was more of an unintended side effect i'd say, and not something we should be testing against or encouraging. if authors want to use BS-style tooltips in their SVG content, they'll need to come up with their own plugin/customisation (for instance, something that tries to find a related `<title>` or `<desc>` element inside the `<svg>`.

note that if authors are using SVG as part of their actual HTML document (for instance, as a graphic inside a `<button>`, then they should add the `title` attribute / hang the tooltip functionality off of the `<button>`, not the `<svg>`.

in short, i'd say this issue is invalid.

**Comment by Johann-S** [comment]
That's ok to me, but should we add an how-to use our Tooltip plugin on svg or something else to our documentation ?

**Comment by mdo** [comment]
Is this still relevant?

**Comment by XhmikosR** [comment]
I guess so, it's what #32715 is about.

**Comment by XhmikosR** [comment]
Or not, I will verify later...

**Comment by XhmikosR** [comment]
Closing as per @patrickhlauke's comment above.
